src/olympia/editors/models.py

```diff
@@ -306,9 +306,13 @@ def base_query(self):
 
     @property
     def authors(self):
+        def uniquify(sq):
+            see = set()
+            return [(x, y) for (x, y) in sq if x not in see and not see.add(x)]
```

Comment:
couldn't just do `set` as its not indexable (the helpers function fails) but can do `list(set(zip(ids, usernames)))`